src-java/grpc-speaker/grpc-service/src/main/java/org/openkilda/grpc/speaker/messaging/MessageProcessor.java

```diff
@@ -107,34 +105,40 @@ private void handleCommandMessage(CommandMessage command, String key) {
     }
 
     private CompletableFuture<Response> handleDumpLogicalPortsRequest(DumpLogicalPortsRequest request) {
-        log.debug("Dumping logical ports on switch {}", request.getAddress());
-        return makeResponse(service.dumpLogicalPorts(request.getAddress())
-                .thenApply(result -> new DumpLogicalPortsResponse(request.getAddress(), result)));
+        CompletableFuture<InfoData> future = service.dumpLogicalPorts(request.getAddress())
+                .thenApply(result -> new DumpLogicalPortsResponse(request.getAddress(), result));
+        return makeResponse(future, String.format("Dumping logical ports on switch %s", request.getAddress()));
+    }
+
+    private CompletableFuture<Response> handleDeleteLogicalPortRequest(DeleteLogicalPortRequest command) {
+        CompletableFuture<InfoData> future = service
+                .deleteConfigLogicalPort(command.getAddress(), command.getLogicalPortNumber())
+                .thenApply(result -> new DeleteLogicalPortResponse(
+                        command.getAddress(), command.getLogicalPortNumber(), result.getDeleted()));
+        return makeResponse(future, String.format(
+                "Delete logical port %s on switch %s", command.getLogicalPortNumber(), command.getAddress()));
     }
 
     private CompletableFuture<Response> handleGetSwitchInfoRequest(GetSwitchInfoRequest request) {
-        log.debug("Getting switch info for switch {}", request.getAddress());
-        return makeResponse(service.getSwitchStatus(request.getAddress())
-                .thenApply(result -> new GetSwitchInfoResponse(request.getAddress(), result)));
+        CompletableFuture<InfoData> future = service.getSwitchStatus(request.getAddress())
+                .thenApply(result -> new GetSwitchInfoResponse(request.getAddress(), result));
+        return makeResponse(future, String.format("Getting switch info for switch %s", request.getAddress()));
     }
 
     private CompletableFuture<Response> handleGetPacketInOutStatsRequest(GetPacketInOutStatsRequest request) {
-        log.debug("Getting switch packet in out stats for switch {}", request.getAddress());
         CompletableFuture<InfoData> future = service.getPacketInOutStats(request.getAddress())
                 .thenApply(stats -> new GetPacketInOutStatsResponse(request.getSwitchId(), responseMapper.map(stats)));
-        return makeResponse(future, statsTopic);
-    }
-
-    private CompletableFuture<Response> handleDeleteLogicalPortRequest(DeleteLogicalPortRequest command) {
-        return makeResponse(service.deleteConfigLogicalPort(command.getAddress(), command.getLogicalPortNumber())
-                .thenApply(result -> new DeleteLogicalPortResponse(
-                        command.getAddress(), command.getLogicalPortNumber(), result.getDeleted())));
+        return makeResponse(
+                future, String.format("Getting switch packet in out stats for switch %s", request.getAddress()),
```

Comment:
You changed `in out` to `in/out` above. Maybe here we should do the same?

Reply:
yes, we should. Updated.